### DISCUSSION TITLE
Cleanup contributors list

### DISCUSSION THREAD

**Discussion by andrasbacsai** [community/community#23966] [New to GitHub]
<p>How can I refresh the contributors list on a repository? It does not reflect reality (see pictures). I look through the commits, but I cannot other contributors, just me.</p>
<p>Repository: <a href="https://github.com/coollabsio/coolify" class="inline-onebox" rel="noopener nofollow ugc">GitHub - coollabsio/coolify: Heroku &amp; Netlify alternative</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/3/d344f0eec0390a1b192a1822626f0aba9050b820.png" data-download-href="/uploads/short-url/u8Ys1q5BtfAVAGp0ngAFtZU7ZQI.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/5845193/181080432-8b720965-e320-475b-998c-be2701ae9056.png" alt="image" data-base62-sha1="u8Ys1q5BtfAVAGp0ngAFtZU7ZQI" width="690" height="281" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d344f0eec0390a1b192a1822626f0aba9050b820_2_690x281.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d344f0eec0390a1b192a1822626f0aba9050b820_2_1035x421.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d344f0eec0390a1b192a1822626f0aba9050b820_2_1380x562.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d344f0eec0390a1b192a1822626f0aba9050b820_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">3620×1476 472 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>I’ve just triggered a contributions rebuild - that should fix it, but it does often take a couple hours.</p>

**Reply by jimmydonaldson** [reply]
I'm having the problem. i recently rewrote the commit history of my repository to correct author information using an interactive git rebase and successfully removed all commits attributed to another individual. After force-pushing the updated history, I confirmed there are no longer any commits or contributions associated with this individual in the repository, both locally and on GitHub. but the repository’s contributor list doesn't accurately reflect the current commit history. how can i fix it? can i trigger a contributions rebuild just like you did?

**Comment by andrasbacsai** [comment]
<p>Wow, thanks! Really appreciate it!</p>

**Comment by ernesteNtezirizaza** [comment]
How can I refresh the contributors list on a repository? I looked through the commits, but I could only see three contributors, but on the list there are four.

![image](https://github.com/user-attachments/assets/e546f124-3f0f-4489-b3ef-ee2bac0bbda0)
